news/2018-05-26-beatmap-spotlights-april-2018.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: post
+title: "Beatmap Spotlights: April 2018"
+date: 2018-05-26 12:00:00 +0000
+---
+
+Bringing awesome maps for every game mode, and a shiny new medal for the collector hiding deep inside you, The April Spotlights are now available!
+
+![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/spotlight-header.jpg)
+
+The beatmap charts for the April Spotlights can be found [here](https://osu.ppy.sh/p/chart?ch=MONTH1805). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/p/packlist?t=r) for easier access. **The April 2018 Spotlights rankings will end on 15th June 2018.**
+
+## Past Spotlights results
+
+First of all, congratulations to the winners of the March 2018 Spotlights. They will receive one month of supporter status for their efforts!
++ **osu!:** [Toy](https://osu.ppy.sh/users/2757689)
++ **osu!taiko:** [-\[ ix Ishida xi \]-](https://osu.ppy.sh/users/242910)
++ **osu!catch:** [Crystal](https://osu.ppy.sh/users/1646397)
+
+# April 2018 Spotlights
+
+## osu!
+
+[![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/iwashi-ga.jpg)](http://osu.ppy.sh/beatmapsets/602367)
+
+Starting off his mapping career with a bang, [Damnjelly](https://osu.ppy.sh/users/1666355) comes onto the scene with his mapset of "Iwashi ga Tsuchi kara Haete Kurunda". The highest difficulty features an escalation style reading challenge, slowly introducing complex rhythms and culminating with overlapping streams. 
```

Comment:
I feel like this could go a bit more in-depth since it's the winner, could talk about the tone the map creates or the hitsounds